stable_baselines3/common/logger.py

```diff
@@ -689,11 +690,14 @@ def configure(folder: Optional[str] = None, format_strings: Optional[List[str]]
     if format_strings is None:
         format_strings = os.getenv("SB3_LOG_FORMAT", "stdout,log,csv").split(",")
 
-    format_strings = filter(None, format_strings)
+    format_strings = list(filter(None, format_strings))
     output_formats = [make_output_format(f, folder, log_suffix) for f in format_strings]
 
     Logger.CURRENT = Logger(folder=folder, output_formats=output_formats)
```

Comment:
Do we still need Logger.CURRENT? It seems confusing to me that we both return a Logger instance but calling `configure` still has side-effects. (Perhaps we can keep Logger.CURRENT but have a deprecation warning when it's used if needed for backward compatibility?)

Reply:
Good point, I removed it as it was not use anymore.